pkg/tfvars/baremetal/baremetal.go

```diff
@@ -155,6 +160,13 @@ func TFVars(libvirtURI, bootstrapProvisioningIP, bootstrapOSImage, externalBridg
 			})
 	}
 
+	var masterIgn igntypes.Config
+	if err := json.Unmarshal([]byte(ignition), &masterIgn); err != nil {
+		return nil, err
+	}
+	ignitionURL := masterIgn.Ignition.Config.Merge[0].Source
+	ignitionURLCACert := masterIgn.Ignition.Security.TLS.CertificateAuthorities[0].Source
```

Comment:
terraform-provider-ignition currently expects just the base64 encoded cert, but this passes the entire dataurl - we need to strip the dataurl prefix here until/unless we rework the provider to accept the full dataurl